### PR TITLE
feat(github): add open in github button

### PR DESCRIPTION
**Pull request by JonasBa**
Adds a "open in github" link to the profiles. We try and fetch the link only when users open a context menu. Will likely need some polishing to do, but we want to determine feasibility first

### CODE REVIEW

File: static/app/components/profiling/flamegraphOptionsContextMenu.tsx [hunk 1]
```diff
@@ -39,12 +45,82 @@ interface FlameGraphOptionsContextMenuProps {
...
+  }, [props.hoveredNode, api, projects, organization, props.profileGroup]);
+
+  const onOpenInGithubClick = useCallback(() => {
+    if (githubLink.type !== 'resolved') {
+      return;
+    }
+    if (!githubLink.data.sourceUrl) {
+      return;
+    }
+    window.open(githubLink.data.sourceUrl, '_blank');
```
Comment (Zylphrex) [inline static/app/components/profiling/flamegraphOptionsContextMenu.tsx:121]: noopener noreferrer?

File: static/app/components/profiling/flamegraphOptionsContextMenu.tsx [hunk 2]
```diff
@@ -39,12 +45,86 @@ interface FlameGraphOptionsContextMenuProps {
...
   onHighlightAllOccurencesClick: () => void;
+  profileGroup: ProfileGroup | null;
+}
+
+function isSupportedPlatformForGitHubLink(platform: string | undefined): boolean {
+  if (platform === undefined) {
+    return false;
+  }
+
+  return /node|python/.test(platform);
```
Comment (Zylphrex) [inline static/app/components/profiling/flamegraphOptionsContextMenu.tsx:56]: Is regex necessary here?

**Review by Zylphrex — APPROVED** [review]

File: static/app/components/profiling/flamegraphOptionsContextMenu.tsx (hunk 2 above)
Comment (JonasBa) [reply static/app/components/profiling/flamegraphOptionsContextMenu.tsx:56]: Looking at the platform keys, it looks like we have node-* and python-* and I just want to enable it for all of those.

Comment (Zylphrex) [reply static/app/components/profiling/flamegraphOptionsContextMenu.tsx:56]: I believe this platform is coming from the profile, not the project so it should be limited to `node` and `python`.

Comment (JonasBa) [reply static/app/components/profiling/flamegraphOptionsContextMenu.tsx:56]: Arf, you are right. I looked at the platform key type and got confused

**Pull request merged**